XPlat/AzureMessaging/Android/source/Xamarin.Azure.NotificationHubs.Android.csproj

```diff
@@ -40,10 +41,20 @@
      <TransformFile Include="Transforms\*.xml" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition=" $(TargetFramework.StartsWith('MonoAndroid90')) ">
     <PackageReference Include="Xamarin.Android.Support.v7.AppCompat" Version="28.0.0.3" PrivateAssets="None" />
-    <PackageReference Include="Xamarin.Firebase.Messaging" Version="71.1740.4" />
+    <PackageReference Include="Xamarin.Google.Android.DataTransport.TransportBackendCct" Version="2.2.2" />
+    <PackageReference Include="Xamarin.Google.Android.DataTransport.TransportRuntime" Version="2.2.2" />
+    <PackageReference Include="Xamarin.Firebase.Messaging" Version="120.1.7" />
```

Comment:
This version depends on AndroidX. I don't think we need to use Android Support anymore. All the libraries used here are AndroidX, so we can drop 90 and make this library AndroidX only.
If the user is still using Android Support, then the upgrade process will migrate everything.

Reply:
Ok, removed all old compat support